addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -53,6 +53,17 @@
 						<bordertexture border="21">overlays/shadow.png</bordertexture>
 						<bordersize>20</bordersize>
 					</control>
+					<control type="image">
+						<left>14</left>
+						<top>4</top>
+						<width>506</width>
+						<height>801</height>
+						<aspectratio>keep</aspectratio>
+						<texture>OverlaySpoiler.png</texture>
+						<bordertexture border="21">overlays/shadow.png</bordertexture>
+						<bordersize>20</bordersize>
+						<visible>String.IsEqual(ListItem.Plot,$LOCALIZE[20370])</visible>
```

Comment:
it would be better use this condition: System.GetBool(videolibrary.showunwatchedplots)

Reply:
This setting is not a bool anymore: 

https://github.com/xbmc/xbmc/blob/a45d319f0ae8b3afde07d1b6b7ed9a77e4e2fd33/system/settings/settings.xml#L971

Reply:
ah, wasn't aware.. ignore :-)